.github/workflows/docker-build-push.yaml

```diff
@@ -65,6 +70,11 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
+    - name: Download an artifact
+      if: ${{ inputs.artifact_name != "" }}
```

Comment:
I think this should be `  if: ${{ inputs.artifact_name != '' }}` - https://github.com/socotra/loadassets2/actions/runs/3534048298

..I don't have access to open a PR in this repo - I would do the change...